### PR TITLE
Example: Add AutoRange SpeedTest Example

### PR DESCRIPTION
**Pull request by dgoeries**
@j9ac9k 

Another example one could add to the package.

### CODE REVIEW

File: examples/PlotLayoutSpeedTest.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+"""
+Update a simple plot as rapidly as possible to measure speed.
+"""
+
+## Add path to library (just for examples; you do not need this)
```
Comment (lidstrom83) [inline examples/PlotLayoutSpeedTest.py:7]: This comment should go above the `import initExample` line

**Comment by pijyoi** [comment]
Compared to the other performance tests, this test limits the time measurement to only the ```curve.setData()``` line. While this is a valid way of measurement, it does mean that the reported ```fps = 1 / average``` does not give the same meaning of including all other overheads needed to redraw the screen.

**Comment by dgoeries** [comment]
I believe i finally found it and rewrote the example to autorange to show the regression between 0.11.0 and master.

**Comment by lidstrom83** [comment]
Can this measurement not be achieved already with PlotSpeedTest.py? For example, by running it and then clicking the auto range button I can detect the performance drop from #1435.

**Comment by dgoeries** [comment]
thanks, yes @lidstrom83 that is correct. I don't intend to merge this one. If you can confirm the performance drop from #1435 i am happy and done.

**Pull request closed without merging**